packages/grid/src/index.scss

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+$grid-gutter-width: 20px !default;
+
+@import '@zendeskgarden/css-variables/dist/index.scss';
+@import 'bootstrap/scss/bootstrap-grid';
+
+/**
+ * RTL locale styling
+ */
+.container.is-rtl, .container-fluid.is-rtl {
+  direction: rtl;
+}
+
+/**
+ * Debug styling for columns
+ */
+.debug {
+  .row > .col, .row > [class^='col-'] {
+    border: 1px solid $zd-color-blue-400;
```

Comment:
Consider applying an alpha 35% in order to smooth out adjoining borders a bit.

Reply:
Added to both background and border.